docs/troubleshooting.md

```diff
@@ -1,19 +1,63 @@
 # Troubleshooting
 
+Start by reviewing the [OpenTelemetry Collector troubleshooting
+documentation](https://github.com/open-telemetry/opentelemetry-collector/blob/master/docs/troubleshooting.md).
+
+## Gathering Support Information
+
+If you are unable to determine why something is not working then you can [email
+support](mailto:signalfx-support@splunk.com). When opening a support request,
+it is important to include as much information about the issue as possible
+including:
+
+- What did you try to do?
+- What happened?
+- What did you expect to happen?
+- Have you found any workaround?
+- How impactful is the issue?
+- How can we produce the issue?
+
+End-to-end architecture information is helpful including:
+
+- What is generating the data?
+- Where was the data configured to go to?
+- What format was the data sent in?
+- How is the next hop configured?
+- Where is the data configured to go from here?
+- What format was the data sent in?
+- Any dns/firewall/networking/proxy information to be aware of?
+
+In addition, it is important to gather support information including:
+
+- Configuration file
+  - Kubernetes: `kubectl get configmap my-configmap -o yaml >my-configmap.yaml`
+  - Linux: `/etc/otel/collector`
+- Logs and ideally debug logs
+  - Docker: `docker logs my-container >my-container.log`
+  - Journald: `journalctl -u my-service >my-service.log`
+  - Kubernetes: `kubectl logs my-pod >my-pod.log`
```

Comment:
On k8s you have to give name of the container if there is more than one. Also adding describe since the events it gives are sometimes useful.

```suggestion
  - Kubernetes: 
      `kubectl describe pod my-pod`
      `kubectl logs my-pod otel-collector >my-pod-otel.log`
      `kubectl logs my-pod fluentd >my-pod-fluentd.log`
```